WalletWasabi.Fluent/ViewModels/Settings/GeneralSettingsTabViewModel.cs

```diff
@@ -38,11 +38,12 @@ public GeneralSettingsTabViewModel()
 			_autoCopy = Services.UiConfig.Autocopy;
 			_customFee = Services.UiConfig.IsCustomFee;
 			_customChangeAddress = Services.UiConfig.IsCustomChangeAddress;
-			_runOnSystemStartup = Services.UiConfig.RunOnSystemStartup;
 			_selectedFeeDisplayFormat = Enum.IsDefined(typeof(FeeDisplayFormat), Services.UiConfig.FeeDisplayFormat)
 				? (FeeDisplayFormat)Services.UiConfig.FeeDisplayFormat
 				: FeeDisplayFormat.SatoshiPerByte;
 
+			_runOnSystemStartup = Services.UiConfig.RunOnSystemStartup = StartupChecker.ValidateAsync().Result;
```

Comment:
This is business logic, you shouldn't do it here. Maybe here somewhere? 
https://github.com/molnard/WalletWasabi/blob/a4c175998b324bc77f1382c1ab141dc6008ab89f/WalletWasabi.Gui/Global.cs#L128

Reply:
That `Global.cs` is for the old Gui. Do we have something similar in the Fluent?

Reply:
You are right, good catch!